backend/app.py

```diff
@@ -386,12 +386,7 @@ def fullfil_request():
                 continue
             DB["inventory"].update_one(
                 {"_id": ObjectId(_id)},
-                {
-                    "$set": {
-                        "itemCount": int(inventory_item["itemCount"])
-                        - 1
-                    }
-                },
+                {"$set": {"itemCount": int(inventory_item["itemCount"]) - 1}},
```

Comment:
[nit](https://onetwobytes.com/2021/11/09/nit-code-review/): try to avoid formatting unrelated to your proposed changes; this is ok though